src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -1,7 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<workspaceFilter version="1.0">
-    <filter root="/apps/${appsFolderName}"/>
-#if ( $optionIncludeErrorHandler == "y" )
-    <filter root="/apps/sling" />
-#end
-</workspaceFilter>
+<?xml version="1.0" encoding="UTF-8"?>
+<workspaceFilter version="1.0">
+    <filter root="/apps/${appsFolderName}"/>
+#if ( $optionIncludeErrorHandler == "y" )
+    <filter root="/apps/sling" />
+#end
+#if ( $isSingleCountryWebsite  == "n" )
+    <filter root="/apps/msm" mode="merge"/>

```

Comment:
This should be more specific - we know what the explicit folder name for this process will be, and we should use it.